template/manifest.json

```diff
@@ -5,9 +5,23 @@
   "description": "Built a plugin!",
   "runtime": "nodejs",
   "main": "index.js",
-  "handlers": {
-    "system.triggered.event": [
-      "my-handler.js"
-    ]
-  }
+  "events": [
```

Comment:
Instead of having `requiredParameters` and `optionalParameters` as separate properties, we should have `parameters` with each value having `required` flag.